src/ec/packages/ec-component-inpage-navigation/inpage-navigation.story.js

```diff
@@ -72,26 +74,37 @@ const btnIdHandler = () => {
   return false;
 };
 
+// Preserve the adapted specs.
+const prepareNavigation = data => {
+  // Buttons for the demo.
+  button(btnLeftLabel, btnLeftHandler, buttonLabels.cases);
+  button(btnMainLabel, btnMainHandler, buttonLabels.cases);
+  button(btnIdLabel, btnIdHandler, buttonLabels.cases);
+  button(btnIdRemoveLabel, btnIdRemoveHandler, buttonLabels.cases);
+
+  data.title = text('Title', data.title, buttonLabels.required);
+  data.links = object('Links', data.links, buttonLabels.required);
+
+  getExtraKnobs(data);
+
+  return data;
+};
+
 storiesOf('Components/Navigation/Inpage navigation', module)
   .addDecorator(withNotes)
   .addDecorator(withKnobs)
   .addDecorator(withCode)
   .add(
     'default',
     () => {
-      // Buttons for the demo.
-      button(btnLeftLabel, btnLeftHandler);
-      button(btnMainLabel, btnMainHandler);
-      button(btnIdLabel, btnIdHandler);
-      button(btnIdRemoveLabel, btnIdRemoveHandler);
-
       let pageFillerHtml = '';
       demoData.links.forEach(content => {
         pageFillerHtml += content.item;
       });
 
       const fullDemoData = { ...demoData, icon_path: iconPath };
```

Comment:
Ahhh, this has to be a knob:
`icon_path`
if you don't set this you don't implement the component correctly, so it's also a mandatory one, a select with a single option...